### ISSUE TITLE
Recommend using Multiply vs Divide in get_micros()

### ISSUE THREAD

**Issue by filmo** [ElectricRCAircraftGuy/eRCaGuy_TimerCounter#3]
On Arduinos [multiply is about 39x faster](https://forum.arduino.cc/index.php?topic=92684.0) than divide. Thus multiplying by 0.50 rather than dividing by 2.0 might make a difference on the get_micros() routine.

**Comment by ElectricRCAircraftGuy** [comment]
yeah I think you're right. Here's some more speed test info: https://playground.arduino.cc/Main/ShowInfo. Really though I recommend you just use get_count() if you want good speed. Avoid the floating point operations altogether. I had get_micros() return a float essentially for those less experienced who are likely using floating point variables and math instead of integer math anyway.

I'll fix it when I get the chance.
